src/routes/transactions/transactions.controller.ts

```diff
@@ -53,13 +53,13 @@ export class TransactionsController {
 
   @ApiOkResponse({ type: MultisigTransactionPage })
   @Get('chains/:chainId/safes/:safeAddress/multisig-transactions')
-  @ApiQuery({ name: 'execution_date__gte', required: false })
-  @ApiQuery({ name: 'execution_date__lte', required: false })
-  @ApiQuery({ name: 'to', required: false })
-  @ApiQuery({ name: 'value', required: false })
-  @ApiQuery({ name: 'nonce', required: false })
-  @ApiQuery({ name: 'executed', required: false })
-  @ApiQuery({ name: 'cursor', required: false })
+  @ApiQuery({ name: 'execution_date__gte', required: false, type: String })
+  @ApiQuery({ name: 'execution_date__lte', required: false, type: String })
+  @ApiQuery({ name: 'to', required: false, type: String })
+  @ApiQuery({ name: 'value', required: false, type: String })
+  @ApiQuery({ name: 'nonce', required: false, type: String })
+  @ApiQuery({ name: 'executed', required: false, type: String })
```

Comment:
We can have `Boolean` as a type for the `executed`. It gives a nice picker in the Swagger interface that way 🙂

<img width="548" alt="image" src="https://github.com/safe-global/safe-client-gateway-nest/assets/3332770/863df090-1bfc-4361-9f02-a8cdac1b0056">